src/NodeJS/Jering.Javascript.NodeJS.csproj

```diff
@@ -97,8 +97,8 @@
         If we aren't multi-targeting, JavascriptBuild runs before PreBuildEvent.
     -->
     <Target Name="JavascriptBuildWindows" BeforeTargets="DispatchToInnerBuilds" Inputs="@(JavascriptInputs)" Outputs="@(JavascriptOutputs)">
-        <Yarn WorkingDirectory=".\Javascript" Command="run build --env mode=$(Configuration) --env entry=./Servers/OutOfProcess/Http/Http11Server.ts" />
-        <Yarn WorkingDirectory=".\Javascript" Command="run build --env mode=$(Configuration) --env entry=./Servers/OutOfProcess/Http/Http20Server.ts" />
+        <Yarn WorkingDirectory=".\Javascript" Command="run build --ssr ./Servers/OutOfProcess/Http/Http11Server.ts --outDir bin/$(Configuration)" />
```

Comment:
Could we add a `<Yarn WorkingDirectory=".\Javascript" />` task here, to trigger the `yarn install`.

Reply:
do you mean to avoid calling yarn install twice? I'm certainly open to that, although I don't think it makes much of a difference as the second yarn install will return very fast. I was trying in the PR to keep the setup as close to the webpack one as possible to make it easier for @JeremyTCD to approve it. Also I'm not an MSBuild expert, so I'm not completely sure what happens if we add another task here, if that would execute sequentially or in parallel.

Reply:
Just seems like the correct way of doing it, but not a biggie if the other way works. :)